core/common/src/RenderMaterial.ts

```diff
@@ -17,16 +18,24 @@ export abstract class RenderMaterial {
   public readonly key?: string;
   /** Describes how to map an image to a surface to which this material is applied. */
   public readonly textureMapping?: TextureMapping;
+  /** Used for ordered comparisons, e.g. in DisplayParams.compareForMerge */
+  private readonly _guid: GuidString;
 
   // eslint-disable-next-line deprecation/deprecation
-  protected constructor(params: RenderMaterial.Params) {
+  protected constructor(params: Pick<RenderMaterial.Params, "key" | "textureMapping">) {
```

Comment:
If `Params` is deprecated (and going to be removed in 5.0?), does it make sense to `Pick` them?

Reply:
addressed.